domains/foxdev.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "GermanFoxDev",
+           "email": "",
+           "discord": "1073620716152434830",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.UbebN16UHO72TKA2YD4x_u4SI0XWphOGenqOawSBEd1FBNohHDWfoJMPcipQ9E-MFGYgvC5gVue7h2sKEoRb6dd4G6v46leW1Tc7_WJAhn55u3b0Rj-fJqjvqcwaNo05W1Zq0O9WfTGQdzXyqVBcSnq1V6GnEeI0UzTCyI_HNqnXzaI1qMmo8CtQgyTKnfBcR1opNCO7QeSG_T4EGg4DLWsrgr0AJO-HVXYvM_h5rCeyQXTfrNAMa9ICAT-TwJGiM-cu-wFBMB_1HQpBFaQaHhH5UAFKIfF9WQQpH-VHf_LhI8FpLRB-6CnCWE5JzPHFVlbpTkVSn20i_oPUrTV4Dg.RyWu6R3v0aMxyponKSI1gg.JEE8XkoGn7xz6tCqENxVjilgS1edQFIWTb38odWEZsF9dwQP_W7-N8nQhyx1oPt5yPxx8Iye9fwgYpeVV_9_bL6ECqHfH-GyPbXSOkygJbk.mDRr1pnHgCfZuEb9BZYs8g"
+        },
+    
+        "record": {
+            "CNAME": "germanfoxdev.is-a.dev"
```

You probably want to redirect this domain to `germanfoxdev.is-a.dev`, this is not how you do it.
You have 2 options
- Use URL records (might not always work
```suggestion
            "URL": "https://germanfoxdev.is-a.dev"
```

- Use a redirect service (more reliable) such as https://redirect.pizza